src/grafana_dashboards/MongoDB_ReplSet_Summary.json

```diff
@@ -931,7 +931,7 @@
             },
             "gridPos": {
                 "h": 8,
-                "w": 24,
+                "w": 8,
```

Comment:
needed?

Reply:
Might not be needed indeed.
I will spin everything up again so I can check